schema/gosling.schema.json

```diff
@@ -428,7 +428,7 @@
           "description": "Determine the colors that should be bound to data value. Default properties are determined considering the field type."
         },
         "type": {
-          "description": "Data type",
+          "description": "Specify the data type",
```

Comment:
Could you double-check this file again? I think this file should be unchanged in this PR.

Reply:
Not very sure about what caused this problem. But the updated description "Specify the data type" is the right description based on the comments in the gosling.schema.ts in this PR.